designer/server/src/test/resources/config/business-cases/tables-definition.sql

```diff
@@ -1,23 +1,22 @@
 CREATE TABLE transactions
 (
-    datatime  TIMESTAMP,
+    datetime  TIMESTAMP,
```

Comment:
woops ;)

Reply:
wasn't datatime a typo? :D In our user story there is datatime, but I don't know what that is

Reply:
yes it was a typo. I just wanted to share my emotional state in this comment. Good catch!